mux_test.go

```diff
@@ -405,16 +405,6 @@ func TestPath(t *testing.T) {
 			pathTemplate: `/{type:(?i:daily|mini|variety)}-{date:\d{4,4}-\d{2,2}-\d{2,2}}`,
 			shouldMatch:  true,
 		},
-		{
```

Comment:
The test started failing if I reverted ```extractVars```. 

I tried making the test pass by modifying the function, but the testcase itself seemed a bit strange imo.

The map ```{"v1": "111", "v2": "", "v3": "222"}``` being the result of the path ```111/222``` seemed funky to me, as there seems to be route parameters missing (shouldn't match at all)

But maybe I'm just missing something